doc/apiv3-documentation.apib

```diff
@@ -3515,6 +3828,20 @@ Gets a list of users that can watch the work package.
                 "message": "You are not authorized to access this resource."
             }
 
++ Response 404 (application/hal+json)
+
+    Returned if the project does not exist or the client does not have sufficient permissions to see it.
+
+    **Required permission:** view project
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:NotFound",
+                "message": "The specified project does not exist."
```

Comment:
This seems weird, I'd have expected this entire response to revolve around the specified _work package_.